app/src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,7 @@
             </intent-filter>
         </activity>
         <service android:name="org.eclipse.paho.android.service.MqttService" />
-
```

Comment:
A tip from serge is that you might need to request legacy storage if you're running API 29.